src/Engines/Hasher/PBKDF2.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace WTSA1\Engines\Hasher;
+
+class PBKDF2 {
+    const algorithm = "pbkdf2_sha256";
+    const iterations = 100000;
+    public static function generate($input, $algorithm = self::algorithm, $iterations = self::iterations, $salt = false) {
+        /**
+        <algorithm>$<iterations>$<salt>$<hash>
+        <algorithm>             We will use Django’s default hashing algorithm: PBKDF2 (SHA256 hash)
+        $                       The dollar sign is used as a separator.
+        <iterations>            The number of algorithm iterations (work factor)
+        <salt>                  The random salt [Best-practice: /dev/urandom)
+        <hash>                  The resulting password hash
```

Comment:
Maybe we should adapt the [phpdoc](https://docs.phpdoc.org/references/phpdoc/basic-syntax.html) syntax for all documentation?🤔 This would make documentation consistent and we could generate a document out of it for the official documentation.

Reply:
@cybrox Good Point 👍 Starting from now, i'm going to use phpdoc for documentation

Reply:
Here are the changes with changed documentation: https://github.com/IBZTI2018/TI3-WT-SA/commit/210d80600c7cbf6ad3318ab1483f5091233aca32